### PR TITLE
add new user blurb template

### PR DESCRIPTION
**Pull request by sarahmccuan**

### CODE REVIEW

File: User-Help/NewUserBlurb.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--- Local Intro Blurb -->
+Welcome to the Writing Codidact community! This is the place for those interested in the craft of writing, editing, and publishing. We welcome questions about all kinds and genres of writing: fiction, poetry, technical documentation, journalism, scriptwriting, non-fiction, essays, and more. For contributing guidelines, check out our site [FAQ's](https://writing.codidact.com/about). 
+
+<!--- Global Intro Blurb -->
+Your contributions make Codidact better! Got questions? Click the "ask" button. Able to offer answers or advice? Use the Lottery button to view a cross section of recent posts, or search for your favorite topics. As you browse, vote on content that stands out - it's how we improve. 
+
+To learn more about the Codidact family of communities and our sister sites, check out [our home page](https://codidact.com). 
+
+<!--- Transition Annoucement -->
+If you have an account on Writing Stack Exchange, check out [this question](https://writing.codidact.com/questions/39219#answer-39220) for instructions on how to claim your questions and answers with you account.
```
Comment (cellio) [inline User-Help/NewUserBlurb.md:10]: Let's replace this with direct instructions:

If you have an account on (site.SE), you can claim your imported questions and answers here. Edit your profile to add the link to your SE profile.  You will now own those posts here on our site, just as if you had posted them directly.

File: User-Help/NewUserBlurb.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!--- Local Intro Blurb -->
```
Comment (cellio) [inline User-Help/NewUserBlurb.md:1]: Suggestion:

Comment (sarahmccuan) [reply User-Help/NewUserBlurb.md:1]: Sorry, your suggestion isn't coming through? not sure if I changed something it was referring to in the latest commit.

Comment (cellio) [reply User-Help/NewUserBlurb.md:1]: Argh -- I didn't notice that my suggested HTML comment got swallowed.  I suggested expanding the comment text to "Local Intro Blurb: Example (adjust for your community)" to make it overly clear that this is an example.

**Comment by ozewski** [comment]
I approve, ready to be merged whenever.

File: User-Help/NewUserBlurb.md (hunk 2 above)
Comment (sarahmccuan) [reply User-Help/NewUserBlurb.md:1]: Oooh, I like that. Let me update.

File: User-Help/GlobalFAQ.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!--- Codidact FAQ -->
```
Comment (cellio) [inline User-Help/GlobalFAQ.md:1]: This is very nice!

File: User-Help/GlobalFAQ.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+# Codidact FAQ
+
+## What is Codidact?
+
+Codidact is an open-source initiative to provide a high-quality, community-controlled, open source Q&A platform. Visit <https://codidact.org> to learn more about the platform we're building to empower anyone to host a Q&A community, or visit <https://codidact.com> to browse our family of hosted sites. 
+
+## Who runs Codidact?
+
+Users just like you! The software is built and hosted by volunteers who are passionate about the need for a community-driven Q&A site. Each site is moderated by a team of dedicated, experienced users who spend their time weeding out spam, coaching newbies, and ensuring questions are on-topic and high-quality.
```
Comment (cellio) [inline User-Help/GlobalFAQ.md:11]: I'd prefer "newcomers" to "newbies" -- less slang (especially for an international audience), and I've been told some find "newbies" a little off-putting.

File: User-Help/GlobalFAQ.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## How do I ask a question?
+
+You will need to register an account. Make sure you are logged into the appropriate topic-related site for your question and click the "Ask Question" button at the top of the page. A template should appear with helpful guidelines on how to ask a quality question. Note that some communities may require you to have a certain trust level before asking a question; always check out the local FAQ page for important information before contributing.
+
+## What are trust levels and how are they calculated?
+<!-- I'm omitting any discussion of reputation for now since consensus seems to be that it is beyond MVP. If that is implemented we can add information here. -->
+Trust levels are the system Codidact uses to determine user privileges. Everyone starts at Trust Level 0 and gradually gains the ability to contribute at a higher level. Trust levels are earned by positively contributing to the site. For more information, see <https://github.com/codidact/docs/wiki/User-Privileges>.
```
Comment (cellio) [inline User-Help/GlobalFAQ.md:19]: Ultimately we'll want doc to be on-site, not on our GitHub wiki, but we don't have that yet.  Can you please file an issue (for later) to sweep the help for github links and replace them with links to help topics (that we will have to create)?  I don't want to lose track of it but we don't need to address it now.  Thanks.

Comment (sarahmccuan) [reply User-Help/GlobalFAQ.md:19]: Yes, I actually have a trust level page on the project board, but it seems this is sufficiently in-flux that we can use the wiki as a source of truth for now and later do something more comprehensive on reputation / trust levels / privileges / etc. Will add issue to review links later.

**Pull request merged**